datumaro/datumaro/components/extractor.py

```diff
@@ -72,6 +72,20 @@ def __eq__(self, other):
 class LabelCategories(Categories):
     Category = namedtuple('Category', ['name', 'parent', 'attributes'])
 
+    @classmethod
+    def from_iterable(cls, iterable):
```

Comment:
I can't see this method really simplifies what's already in the class constructor. How about providing an option to pass just a list labels in `Dataset.from_iterable()`?

Reply:
There is such opportunity. You can pass a list labels' names or a list with labels' params. I thing it's more flexible and allow you to simplify more tests.